go/signedexchange/bigendianint.go

```diff
@@ -18,6 +18,9 @@ func Encode3BytesBigEndianUint(n int) ([3]byte, error) {
 	}, nil
 }
 
-func Decode3BytesBigEndianUint(b [3]byte) int {
+func Decode3BytesBigEndianUint(b []byte) int {
```

Comment:
No strong reason. I'd want to avoid copying arrays and pass slices instead. Perhaps `Encode3Byte...` should be fixed in the same way for consistency, or we should keep `[3]byte` here for consistency. I prefer slices, but both are fine to me.

Reply:
Let's keep it `[3]byte` then, or please provide reference to coding standards, etc.

Reply:
Done.